server/api/member_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/pd/server"
+)
+
+func TestJoin(t *testing.T) {
+	TestingT(t)
+}
+
+var _ = Suite(&testMemberAPISuite{})
+
+type testMemberAPISuite struct {
+	hc *http.Client
+}
+
+func (s *testMemberAPISuite) SetUpSuite(c *C) {
+	s.hc = newHTTPClient()
+}
+
+func checkListResponse(c *C, body []byte, cfgs []*server.Config) {
+	got := []memberInfo{}
+	json.Unmarshal(body, &got)
+
+	c.Assert(len(got), Equals, len(cfgs))
+
+	for _, memb := range got {
+		ok := false
+		for _, cfg := range cfgs {
+			if memb.Name == cfg.Name {
+				mcu := memb.ClientUrls
```

Comment:
please use readable variable names, what is for `mcu`, `ccu`? I even see `cpu` below.